### PR TITLE
Support for LLMonitor - open-source observability & analytics

### PR DESCRIPTION
**Pull request by vincelwt**
Adds support for [llmonitor](https://llmonitor.com). Tested with errors, chat completion, embeddings (on cohere and openai).
It enables:
- Requests tracking / logging / analytics
- Error debugging
- Cost analytics
- User tracking
 
We will update the [docs](https://llmonitor.com) on our side with instructions for litellm once this is merged and live.

Sorry for all the diff lines, seems like my formatter was slightly different. 

Let me know if anything needs to be changed for merge.

Thank you!

### CODE REVIEW

File: docs/my-website/docs/observability/llmonitor_integration.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# LLMonitor Tutorial
+
+[LLMonitor](https://llmonitor.com/) is an open source observability platform that provides cost tracking, user tracking and powerful agent tracing.
```
Comment (ishaan-jaff) [inline docs/my-website/docs/observability/llmonitor_integration.md]: @vincelwt can you add a SS of what the llmonitor interface looks like ?

**Review by ishaan-jaff — APPROVED** [review]
LGTM ! I'd recommend adding a ss of llmonitor to our docs - so users know what it's like 

Even a live demo dashboard would be super useful

**Comment by ishaan-jaff** [comment]
@vincelwt looks like there are some merge conflicts too

**Comment by ishaan-jaff** [comment]
💥 💥 💥 💥 💥 💥 💥 💥 💥

**Comment by vincelwt** [comment]
@ishaan-jaff Good idea! Added our quick demo video and resolved the merge conflicts :) Should be good to go 🎉

**Pull request merged**